oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/authentication/JwtGrantedAuthoritiesConverterTests.java

```diff
@@ -139,6 +139,59 @@ public void convertWhenTokenHasEmptyScopeAndNonEmptyScpThenScopeAttributeIsTrans
 		assertThat(authorities).isEmpty();
 	}
 
+	@Test
+	public void convertWhenTokenHasEmptyScopeAndEmptyScpAttributeThenTranslatesToNoAuthorities() {
```

Comment:
It looks like some of these tests are meant to improve the overall coverage. Since they aren't directly related to the enhancement, will you please put them in a separate commit, in this same PR?

Reply:
Yes, these are meant to improve test coverage. Will move these into separate commit.